src/core/manager.c

```diff
@@ -3934,6 +3936,19 @@ static int build_generator_environment(Manager *m, char ***ret) {
                         return r;
         }
 
+        cv = detect_confidential_virtualization();
+        if (cv < 0)
+                log_debug_errno(cv, "Failed to detect confidential virtualization, ignoring: %m");
+        else if (v > 0) {
+                const char *s;
+
+                s = strjoina(confidential_virtualization_to_string(cv));
```

Comment:
the strjoina() serves no purpose here, given you have a single argument only (it's supposed to concat strings, that's all)

Reply:
Yep, copy/paste leftover from the code above.